app/modules/transcribe/overlay/keypad-slide.directive.js

```diff
@@ -48,6 +47,25 @@ function keypadSlide(hotkeys, overlayConfig) {
             }
             textarea.caret(startTag.length + text.length);
         }
+
+        function preLoad() {
+            var deferred = $q.defer();
+            var imageArray = [];
+            for (var i = 0; i < $scope.abbreviations.length; i++) {
+                imageArray[i] = new Image();
+                imageArray[i].src = $scope.abbreviations[i].imgPath;
+            }
+            imageArray.forEach.onload = function () {
```

Comment:
Also, a thought - you probably want this to kick off the preload as soon as the page loads, so there's no delay when opening up the keypad. The alternative could be to require it to complete in a `resolve` in ui-router, but that can come later...